core/src/main/scala/ai/h2o/sparkling/backend/external/ExternalH2OBackend.scala

```diff
@@ -96,7 +96,7 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Shell
   private def getExternalH2ONodesArguments(conf: H2OConf): Seq[String] = {
     // Application tags shown in Yarn Resource Manager UI
     val yarnAppTags = s"${ExternalH2OBackend.TAG_EXTERNAL_H2O},${ExternalH2OBackend.TAG_SPARK_APP.format(hc.sparkContext.applicationId)}"
-    new ArgumentBuilder()
+    val commonArguments = new ArgumentBuilder()
       .add(Seq(conf.externalHadoopExecutable, "jar", conf.h2oDriverPath.get))
```

Comment:
NIT: We could do: 
`.add(getHiveArguments(conf), conf.isHiveSupportEnabled)`

Reply:
Good idea, will do. Thanks!